packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -1,68 +1,54 @@
-import { useEffect, useMemo } from 'react'
-import { useAccount } from 'wagmi'
 import { useLocalStorage } from '@uidotdev/usehooks'
+import { Tab } from '@headlessui/react'
+import { create } from 'zustand'
 
 import { TransferPanel } from '../TransferPanel/TransferPanel'
-import { SidePanel } from '../common/SidePanel'
-import { useAppContextActions, useAppContextState } from '../App/AppContext'
 import { ArbitrumStats, statsLocalStorageKey } from './ArbitrumStats'
 import { SettingsDialog } from '../common/SettingsDialog'
 import { TransactionHistory } from '../TransactionHistory/TransactionHistory'
-import { useTransactionHistory } from '../../hooks/useTransactionHistory'
-import { isTxPending } from '../TransactionHistory/helpers'
-import { TransactionStatusInfo } from '../TransactionHistory/TransactionStatusInfo'
+import { TopNavBar } from '../TopNavBar'
 
-function TransactionHistorySidePanel() {
-  const { closeTransactionHistoryPanel } = useAppContextActions()
-  const {
-    layout: { isTransactionHistoryPanelVisible }
-  } = useAppContextState()
-  const { address } = useAccount()
-
-  const transactionHistoryProps = useTransactionHistory(address, {
-    runFetcher: true
-  })
-
-  const { transactions, updatePendingTransaction } = transactionHistoryProps
-
-  const pendingTransactions = useMemo(() => {
-    return transactions.filter(isTxPending)
-  }, [transactions])
-
-  useEffect(() => {
-    const interval = setInterval(() => {
-      pendingTransactions.forEach(updatePendingTransaction)
-    }, 10_000)
-
-    return () => clearInterval(interval)
-  }, [pendingTransactions, updatePendingTransaction])
+enum MainContentTabs {
+  Bridge = 0,
+  TransactionHistory = 1
+}
 
-  return (
-    <SidePanel
-      isOpen={isTransactionHistoryPanelVisible}
-      onClose={closeTransactionHistoryPanel}
-      scrollable={false}
-      panelClassNameOverrides="pb-8"
-    >
-      <TransactionHistory props={{ ...transactionHistoryProps, address }} />
-    </SidePanel>
-  )
+type MainContentTabStore = {
+  selectedTab: MainContentTabs
+  setSelectedTab: (index: MainContentTabs) => void
+  openBridge: () => void
```

Comment:
Could we rename these to `switchToBridge` and `switchToTransactionHistory` to be consistent with what we have in E2E's? Plus it sounds better since we aren't opening any panel - just switching to it.